test/unit/controllers/todo.controller.test.ts

```diff
@@ -0,0 +1,181 @@
+import { expect } from '@loopback/testlab';
+import { TodoController } from '../../../src/controllers';
+import {
+  TodoRepository,
+  TodoItemRepository
+} from '../../../src/repositories/index';
+import * as sinon from 'sinon';
+import { Todo } from '../../../src/models/index';
+
+describe('TodoController', () => {
+  let todoRepo: TodoRepository;
+  let todoItemRepo: TodoItemRepository;
+
+  function resetRepositories() {
+    todoRepo = sinon.createStubInstance(TodoRepository);
+    todoItemRepo = sinon.createStubInstance(TodoItemRepository);
+  }
+  beforeEach(resetRepositories);
+  const payload = new Todo();
+  payload.title = 'do a thing';
+  payload.desc = 'There are some things that need doing';
+  payload.isComplete = false;
+  const payloadWithId = Object.assign({}, payload, { id: 1 });
```

Comment:
Please apply all my comments for `todo-item.controller.test.ts` to this test file too.